### ISSUE TITLE
Caption - refactor caption title label

### ISSUE THREAD

**Issue by wilkus27** [openstax/cnx-recipes#4626]
- Change component name from Caption__Title--Label to Caption__TitleLabel.
- Carnival - create one component (now there are separate components with the same properties for figures, tables and chapter intro)
- Remove unnecessary settings from Astronomy `book.scss`

**Comment by wilkus27** [comment]
@lwtchu In Carnival in Figures and Tables caption label is uppercase, but not in a Chapter Intro. This is how it looks now:

![image](https://user-images.githubusercontent.com/40228854/165109607-5d71ba59-a201-4237-b781-0ae134b6928a.png)

![image](https://user-images.githubusercontent.com/40228854/165109668-defe7950-8a1d-4940-8f4e-3fa19e72c8c7.png)

![image](https://user-images.githubusercontent.com/40228854/165109782-2f4384ac-c078-4a73-9fe2-7378db6499a3.png)

I added uppercase everywhere (in Carnival) for consistency. That's how it was done also in Cosmos. Is it ok? Or should I keep lowercase in Chapter Intro?

**Comment by lwtchu** [comment]
@wilkus27 this is ok - we can keep them all upper case.

**Comment by MartaSztuka** [comment]
#### PASSED
###### chemistry
![Screenshot from 2022-04-27 14-33-02.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/ab1feec1-71b5-4da1-b3b1-e57fea1cd4e2)
![Screenshot from 2022-04-27 14-33-10.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/7b26d675-4f3b-4115-a001-b41306f999d0)
![Screenshot from 2022-04-27 14-33-23.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/29945973-71b6-4225-95f8-f23b35475023)
###### chemistry af
![Screenshot from 2022-04-27 14-48-53.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/a345639b-87de-47e5-9ac6-10f19eb038be)
![Screenshot from 2022-04-27 14-49-11.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/91e33178-b136-409d-8ff4-1d4d96233348)
![Screenshot from 2022-04-27 14-49-18.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/03feb26a-2a09-46dc-808a-9a68e04be9be)
###### political science
![Screenshot from 2022-04-27 12-57-08.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/b090406b-397f-4a0b-ac26-3edfe5885437)
![Screenshot from 2022-04-27 12-59-38.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/6387cfef-c0a9-40c2-b81a-bbb5221676a1)
![Screenshot from 2022-04-27 12-59-44.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/04085af3-fac5-47ca-b118-c89d899ca311)
###### philosophy
![Screenshot from 2022-04-27 13-19-38.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/a13ec4a3-01aa-407a-a4a0-cc20b17fe8fb)
![Screenshot from 2022-04-27 13-19-53.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/8fc5a599-f5fd-4964-b745-1eb2caf21e84)
![Screenshot from 2022-04-27 13-20-31.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/80ea1d7b-812c-4b93-81b4-834fda91f96a)
###### anatomy&phisiology
![Screenshot from 2022-04-27 14-51-14.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/fc126e59-f713-48e9-901e-86cc1b1ad128)
![Screenshot from 2022-04-27 14-51-44.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/b27e2462-2711-4241-9cbd-062339c2b257)
![Screenshot from 2022-04-27 14-52-27.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/65bd2af7-8a39-4ed9-a990-c547461930d7)
###### anthropology
![Screenshot from 2022-04-27 15-06-39.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/3548ff3a-9fa0-4674-83f3-3b4e7149a35f)
![Screenshot from 2022-04-27 15-06-52.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/7b290c99-0149-4ea5-bc84-73922668730c)
![Screenshot from 2022-04-27 15-07-26.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/ae09949b-daa6-4bf5-9e12-657b58a16af0)
###### writing guide
![Screenshot from 2022-04-27 15-16-26.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/4e7ef905-14e7-4d1e-8f89-998d38eaaada)![Screenshot from 2022-04-27 15-16-46.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/9c123657-6216-4c91-baff-243a6c0da8ca)![Screenshot from 2022-04-27 15-17-00.png](https://images.zenhubusercontent.com/5cb5ec14fec3b63424e0f03c/a0bf0564-4ec3-4c61-b06f-bf83ca477dd4)

**Comment by lwtchu** [comment]
![Screen Shot 2022-05-09 at 3.12.40 PM.png](https://images.zenhubusercontent.com/5b57310fb26dd832dfe45b43/fe1bebbf-e27b-4a7f-b5ec-c03383d7532c)
![Screen Shot 2022-05-09 at 3.13.03 PM.png](https://images.zenhubusercontent.com/5b57310fb26dd832dfe45b43/a588a703-fe14-489a-a190-e13090655ed8)

US History

![Screen Shot 2022-05-09 at 3.13.28 PM.png](https://images.zenhubusercontent.com/5b57310fb26dd832dfe45b43/cf2c24bf-6c68-4cbd-915a-1a900957facf)
![Screen Shot 2022-05-09 at 3.13.44 PM.png](https://images.zenhubusercontent.com/5b57310fb26dd832dfe45b43/dc7ef2c4-55df-47d5-8b9d-1025bd5ef05b)
uni physics v 3

@wilkus27 this wasn't done for cardboard or corn right?

**Comment by wilkus27** [comment]
@lwtchu No, upper case was done only in Carnival and Cosmos. Should I applied it to Cardboard and Corn as well?
